src/expandable-section/__tests__/expandable-section.test.tsx

```diff
@@ -323,16 +329,22 @@ describe('Variant container with headerText', () => {
     expect(content).toHaveAttribute('aria-label', 'ARIA Label');
     expect(headerButton).not.toHaveAttribute('aria-labelledby');
   });
-  test('set aria-labelledby when no headerAriaLabel is set', () => {
+  test('set aria labels when no headerAriaLabel is set', () => {
+    const wrapper = renderExpandableSection({
+      variant: 'container',
+      headerText: 'Header component',
+    });
+    const headerButton = wrapper.findHeader().find('[role="button"]')!.getElement();
+    expect(headerButton).toHaveAccessibleName('Header component');
+  });
+  test('set aria description if description present', () => {
     const wrapper = renderExpandableSection({
       variant: 'container',
       headerText: 'Header component',
-      headerCounter: '(5)',
```

Comment:
do we now completely exclude counter from the announcement?

Reply:
From the button, yes, but given that the counter is no longer (since https://github.com/cloudscape-design/components/pull/1143) actually part of the clickable button area I think this makes more sense. It's still part of the heading's accessible name